src/Middleware/RateLimiting/src/DisableRateLimitingAttribute.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// Metadata that disables request rate limiting on an endpoint.
```

Comment:
"Completely disables the rate limiting middleware from applying to this endpoint."